core/tests/webdriverio_utils/LibraryPage.js

```diff
@@ -222,7 +222,7 @@ var LibraryPage = function () {
     var explorationCard = await allExplorationsTitled(explorationName)[0];
     // The Exploration summary card is masked by a dummy element. Therefore, a
     // Javascript click is used.
-    await action.click('Exploration Card', explorationCard, true);
```

Comment:
why don't we need to use a JS click here anymore? It doesn't look like we've changed whether the exploration card is masked